modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -79,8 +79,38 @@ For this reason, data access time is the same, whether you store keys in one par
 
 == How long can data persist in Object Store v2?
 
-Object Store v2 data persists 30 days. The expiry time is reset if an update to that key occurs. If there is no update, the key is deleted at the 30 day mark.
+The maximum TTL (time to live) is 2,592,000 seconds (30 days).
 
+The `entryTtl` value, specified in the global configuration parameters for the connector,
+determines when to evict key-value pairs from the store.
+
+The TTL can be either _rolling_ or _static_:
+
+Rolling::
+Accessing the data extends the TTL for 30 days.
++
+The minimum rolling TTL is 1 second.
+The maximum rolling TTL is unlimited as long as the data continues to be accessed.
+Static::
+The value of `entryTtl` determines when the data is evicted.
++
+The minimum static TTL is 1 second.
+The maximum (and default) static TTL is 2592000 seconds (30 days).
```

Comment:
see here, no commas
GLOBALLY standardize or make the distinction between the amount (commas) and the syntax value (none).